website/public/assets/datadog-dashboard.json

```diff
@@ -145,7 +145,41 @@
         "size": "sm",
         "has_uniform_y_axes": true
       },
-      "layout": { "x": 0, "y": 5, "width": 12, "height": 7 }
+      "layout": { "x": 0, "y": 5, "width": 12, "height": 3 }
+    },
+    {
+      "id": 4618294854537207,
+      "definition": {
+        "title": "Throttled APIs detected in logs over time",
+        "title_size": "16",
+        "title_align": "left",
+        "show_legend": false,
+        "legend_layout": "auto",
+        "legend_columns": ["avg", "min", "max", "value", "sum"],
+        "type": "timeseries",
+        "requests": [
+          {
+            "formulas": [{ "formula": "a" }],
+            "queries": [
+              {
+                "data_source": "logs",
+                "name": "a",
+                "indexes": ["*"],
+                "compute": { "aggregation": "count" },
+                "group_by": [],
+                "search": {
+                  "query": "(@retry.attempt:>=2 OR \"retrying request\") @service.name:$service.value"
```

Comment:
`retrying request` query works on AWS and all plugins that use https://github.com/hashicorp/go-retryablehttp with https://github.com/cloudquery/plugin-sdk/blob/25ed3d25a529a22f351ab92e22fb03a19c9557d4/helpers/retryablehttp/logger.go

`@retry.attempt:>=2` works out of the box for GCP since we send a `retry.attempt` as a log field.

For Azure we need a log pipeline added in https://github.com/cloudquery/integrations-extras/blob/47a83adf4d7aa921a2fd432a676a059ce436fdee/cloudquery/assets/logs/cloudquery.source.azure.yaml#L18